src/System.Collections/tests/Configurations.props

```diff
@@ -2,7 +2,6 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp;
```

Comment:
From a quick look we do have https://github.com/dotnet/corefx/blob/master/src/System.Collections/tests/System.Collections.Tests.csproj#L12 which we will drop if you remove this configuration.